### DISCUSSION TITLE
My account is marked as spam

### DISCUSSION THREAD

**Discussion by abbasi303** [community/community#72866] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello , Can you kindly help me solve my account problem. I have been marked as spam no idea why
And i dont know how to unspam

**Comment by 14AMAN** [comment]
Hey,
My account got marked as Spam too, did it restore?
How many days it took and the steps?

**Comment by cehfisher** [comment]
Hi @abbasi303 - It looks like your issue was mistakenly added to the accessibility discussion board. @queenofcorgis could you please move this to the correct category? Thanks!

**Comment by 14AMAN** [comment, marked as answer]
Raise a ticket regarding your issue, most probably in 2-4 weeks yous
problem will be solved.
As mine was solve too, till then you can make new GitHub account and import
your previous repositories.

**Reply by HimanshuW4** [reply]
@14AMAN have you received any email or something like that, when your account was marked as **Spammy** or when your account is back to normal? It happened to me recently, but haven't received any email from GitHub about it. I have raised the tickets, let's see when it will be unmarked as **Spammy**.

**Reply by Rahman1-You** [reply]
why is there need of creating new account when you will recover your previous one?

**Comment by cehfisher** [comment]
Thanks for posting in the GitHub Community, @abbasi303 and responding @14AMAN! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've asked @queenofcorgis to move this issue into the correct category for you. Good luck!

**Comment by shivam0p** [comment]
Hey! My account is also marked as spam although i just created my account 1 day ago please help to solve it.

**Comment by ErfanMomeniii** [comment]
hi @cehfisher, would you be able to help me? my account was marked as spam.
I have also created a ticket for that but I didn't receive any response.
Ticket: https://support.github.com/ticket/personal/0/2979214

Thanks in advance.

**Comment by Taj-2006** [comment]
My account is marked as spam i just created my account 2days ago

**Reply by uche102** [reply]
same thing happened to my account .

**Comment by AbdelrahmanNazeh748** [comment]
Hi , my account marked as spam too 
what is my problem ?

**Comment by MmahmoudEg** [comment]
i have the same problem

**Comment by Huseynov-Huseyn** [comment]
Hello GitHub Support Team,

I have noticed that my account has been flagged as spam. I believe this is a mistake, as I have not engaged in any activities that violate GitHub's policies. Could you please review my account and remove the spam flag?

Thank you for your assistance.

Best regards,
Huseyn Huseynov

**Comment by ladygalactica** [comment]
Hi, pleaase help, my account is flagged as spam, my fiance who is a programmer is trying ro get me to learn. helo plz

**Comment by redshark1000** [comment]
Hey, my account has also been marked as spam, is there an issue, I've done little to no activitey on this account.
